src/CMakeLists.txt

```diff
@@ -10,7 +10,11 @@ endif()
 ################################
 project( geosx LANGUAGES C CXX )
 set( BLT_CXX_STD "c++14" CACHE STRING "Version of C++ standard" )
-set( ENABLE_WARNINGS_AS_ERRORS "ON" CACHE PATH "" )
+if(CMAKE_BUILD_TYPE EQUAL "Debug")
+  set( ENABLE_WARNINGS_AS_ERRORS "OFF" CACHE PATH "")
+else()
+  set( ENABLE_WARNINGS_AS_ERRORS "ON" CACHE PATH "")
+endif()
```

Comment:
It is intentional. I had some annoying warnings on the Mac, and we have the suppression of `WARNINGS_AS_ERROS` in the `GeosxOptions.cmake` file, but it didn't seem to work for the Mac because they were compiler specific options. I just thought that this approach is the better way as it tells `BLT` that we don't want warnings as errors for debug builds from the start instead of trying to remove it after it has ben set already.